src/prefect/utilities/logging.py

```diff
@@ -32,147 +32,146 @@
     "task_run_id",
 )
 
+MAX_LOG_LENGTH = 1_000_000  # 1 MB - max length of a single log message
+MAX_BATCH_LOG_LENGTH = 50_000_000  # 50 MB - max total batch size for log messages
```

Comment:
cc @cicdw, can you confirm these numbers are sufficient with cloud?

Note that the max size that can be sent in a batch is actually `MAX_BATCH_LOG_LENGTH + MAX_LOG_LENGTH - 1`. Since we can't (easily) peek to see if the next entry will exceed the total batch size, we batch all logs until empty or the limit is exceeded (since we check earlier to prevent entries larger than `MAX_LOG_LENGTH`, that's the max a single log can increment the total).

Reply:
Yea these are compatible with Cloud, although 50MB seems large to me (I know there's a small chance of hitting it, but I wouldn't be surprised if we reduced our payload limits at some point).

Reply:
What batch size would make more sense? 20 MB? 10?

Reply:
I vote 20